test/known_issues/test-stdin-is-always-net.socket.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const spawn = require('child_process').spawn;
+
+if (process.argv[2] === 'child') {
+  // Refs: https://github.com/nodejs/node/pull/5916
+  assert(process.stdin instanceof net.Socket);
```

Comment:
Ugh, I copied from rich's test. Yeah, it probably needs to check the stdout on fail, or something.